client/components/forms/docs/example.jsx

```diff
@@ -77,10 +82,14 @@ var FormFields = React.createClass( {
 
 					<FormFieldset>
 						<FormLegend>Form Checkbox</FormLegend>
-						<FormLabel>
-							<FormCheckbox id="comment_like_notification" name="comment_like_notification" />
-							<span>Email me when someone Likes one of my comments.</span>
-						</FormLabel>
+
+						<Checkbox
+							id="comment_like_notification"
```

Comment:
I see your point, but I tried to be explicit. You are right about the `id` but `name` attribute is used when sending data in a form submission. Considering this is a form field, I tend to include it always.